_posts/2023-07-10-custom-http-signaling-notification.adoc

```diff
@@ -0,0 +1,252 @@
+`io.debezium.pipeline.notification.channels.NotificationChannel` files respectively.
+
+Compile and export the Java project as a JAR file. This can be done using Maven or your preferred build tool. Copy the JAR file to the directory that contains the JAR file for the Debezium connector you want to use. For example, if you want to use the custom signal and notification channels with the Debezium Postgres connector, copy the JAR file to the `/kafka/connect/debezium-connector-postgres` directory.
+
+This example provides a Docker Compose file with necessary services defined including the Mock Server, Zookeeper, Kafka Connect, and Postgres database.
+
+To start the services, run the following command:
+
+[source,shell]
+----
+export DEBEZIUM_VERSION=2.3
+docker-compose up -d
+----
+
+After ensuring that the services are up and running, and the Postgres database is ready to accept connections, the next step is to register the connector. This involves creating a connector configuration file. Let's create a file named `register-postgres.json` with the following properties:
+
+[source,json]
+----
+{
+  "name": "inventory-connector",
+  "config": {
+    "connector.class": "io.debezium.connector.postgresql.PostgresConnector",
+    "tasks.max": 1,
+    "database.hostname": "postgres",
+    "database.port": 5432,
+    "database.user": "postgres",
+    "database.password": "postgres",
+    "database.dbname" : "postgres",
+    "topic.prefix": "dbserver1",
+    "schema.include.list": "inventory",
+    "signal.enabled.channels": "http", // <1>
+    "notification.enabled.channels": "http" // <2>
+  }
+}
+----
+
+<1> The `signal.enabled.channels` property specifies the signal channel to be used by the connector. In this case, the connector uses the `http` signal channel.
+<2> The `notification.enabled.channels` property specifies the notification channel to be used by the connector. In this case, the connector uses the `http` notification channel.
+
+Now that we have the connector configuration file prepared, we can proceed to register the connector with Kafka Connect by executing the following command:
+
+[source,shell]
+----
+curl -i -X POST -H "Accept:application/json" \
+    -H  "Content-Type:application/json" http://localhost:8083/connectors/ \
+    -d @register-postgres.json
+----
+
+Once the connector is successfully registered, you can review the connector logs to observe the signal events. The logs provide insights into the processing and progress of the connector, including any signal-related information. You will encounter log messages similar to the following:
+
+[source,shell]
+----
+http-signaling-notification-connect-1     | 2023-07-07 15:09:48,205 INFO   Postgres|dbserver1|postgres-connector-task  Recorded signal event 'SignalRecord{id='924e3ff8-2245-43ca-ba77-2af9af02fa07', type='log', data='{"message":"Signal message received from http endpoint."}', additionalData={}}'    [io.debezium.examples.signal.HttpSignalChannel]
```

Comment:
```suggestion
Recorded signal event 'SignalRecord{id='924e3ff8-2245-43ca-ba77-2af9af02fa07', type='log', data='{"message":"Signal message received from http endpoint."}', additionalData={}}'    [io.debezium.examples.signal.HttpSignalChannel]
```
The prefix bit is a bit verbose and doesn't really provide much value from the blog's intent, so I'd suggest removing it and starting with the relevant log text.